spec/integration/inventory2_spec.rb

```diff
@@ -282,7 +287,10 @@ def fact_plan(name = 'facts_test')
       expect(result['existing_vars']).to eq(expected_hash_pre)
       expect(result['added_facts']).to eq(expected_hash_post)
       expect(result['added_vars']).to eq(expected_hash_post)
-      expect(result['target_from_string']).to eq(["Target('bar_1', {})"])
+      expect(result['target_not_overwritten']).to eq("dont_overwrite")
+      # TODO, Given bar_1 is defined by just a name, printing the target gets an empty string.
```

Comment:
The alternative would be to give this a uri above.

Reply:
Cool. That works for me.